unison-core/src/Unison/Name.hs

```diff
@@ -51,8 +52,11 @@ sortNames :: [Name] -> [Name]
 sortNames = sortNamed id
 
 sortNamed :: (a -> Name) -> [a] -> [a]
-sortNamed by as = let
-  as' = [ (a, Text.unpack (toText (by a))) | a <- as ]
+sortNamed by = sortByText (toText . by)
+
+sortByText :: (a -> Text) -> [a] -> [a]
```

Comment:
I'm trying to decide whether these two would benefit from a short haddock, or if the signature is good enough.  I can't decide.